### PR TITLE
{lib}[GCCcore/5.4.0] zstd v1.3.4

### PR DESCRIPTION
**Pull request by vanzod**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/l/lz4/lz4-1.8.2-GCCcore-5.4.0.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+source_urls = ['https://github.com/lz4/lz4/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['0963fbe9ee90acd1d15e9f09e826eaaf8ea0312e854803caf2db0a6dd40f4464']
+
+builddependencies = [('binutils', '2.26')]
+
+skipsteps = ['configure']
+
+maxparallel = 4
```
Comment (boegel) [inline easybuild/easyconfigs/l/lz4/lz4-1.8.2-GCCcore-5.4.0.eb]: @vanzod Any specific reason for this?

Comment (vanzod) [reply easybuild/easyconfigs/l/lz4/lz4-1.8.2-GCCcore-5.4.0.eb]: Yes, there are few issues filed for high parallel make. It looks like they managed to get it up and running correctly up to `6` threads, so I set `4` for safety.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (3 easyconfigs in this PR)
node2646.swalot.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/714a211714e60803c5d821092322a047 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2056.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/f0f32daf449897d1136c02eccf9cbe66 for a full test report.

File: easybuild/easyconfigs/l/lz4/lz4-1.8.2-GCCcore-5.4.0.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/l/lz4/lz4-1.8.2-GCCcore-5.4.0.eb]: Is it worth including a reference to those known issues as a comment above this line?

Comment (vanzod) [reply easybuild/easyconfigs/l/lz4/lz4-1.8.2-GCCcore-5.4.0.eb]: Does the value of `parallel` affects also the `make test`?

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
cermis - Linux debian 9.4, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.13
See https://gist.github.com/73a56b06c9e59d79b49a34b3384f1b58 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2646.swalot.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/027c2a921f06c3f151a4f28ee6fa3f2a for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2056.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/e125a5942c2b3fb445db17ab48147cd7 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @vanzod!

**Pull request merged**